test/parallel/test-repl-missing-parenthesis-recoverable.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+/*
```

Comment:
JavaScript the Good Parts says

> The `/* */` form of block comments came from a language called PL/I. PL/I chose
those strange pairs as the symbols for comments because they were unlikely to occur
in that language’s programs, except perhaps in string literals. In JavaScript, those
pairs can also occur in regular expression literals, so block comments are not safe for
commenting out blocks of code

I am neutral about this though.